python/cuml/cuml/manifold/simpl_set.pyx

```diff
@@ -331,7 +333,7 @@ def simplicial_set_embedding(
     umap_params.repulsion_strength = <float> gamma
     umap_params.negative_sample_rate = <int> negative_sample_rate
     umap_params.n_epochs = <int> n_epochs
-    umap_params.random_state = <int> random_state
+    umap_params.random_state = <uint64_t> random_state
```

Comment:
Can you briefly comment on the motivation for this change?

Reply:
The `random_state` is a `uint64_t` in the UMAP parameters struct. Also, there was sometimes a bug when not providing a seed as Cython does not want to set a negative value for a `uint64_t` it seems.